src/topk.c

```diff
@@ -167,26 +167,23 @@ size_t TopK_Count(TopK *topk, const char *item, size_t itemlen) {
     Bucket *runner = NULL;
     uint32_t fp = TOPK_HASH(item, itemlen, GA);    
     // TODO: The optimization of >heapMin should be revisited for performance
-    counter_t heapMin = checkExistInHeap(topk, item, itemlen) ? topk->heap->count : 0;
+    counter_t heapMin = topk->heap->count;
+    HeapBucket *heapPtr = checkExistInHeap(topk, item, itemlen) ? topk->heap->count : 0;
```

Comment:
Compiler warning:
`warning: initialization makes pointer from integer without a cast`

What is supposed to be happening here?

Reply:
I'm using the pointer as a flag to know if number exist in heap

Reply:
Great point. just saving the point and later checking whether it is NULL.